packages/terra-date-picker/src/DateUtil.jsx

```diff
@@ -142,6 +143,103 @@ class DateUtil {
     return undefined;
   }
 
+  /**
+   * Gets the date format order from moment's long date format.
+   * @param {string} momentDateFormat - Moment's long date format.
+   * @return {string} - The date format order. Possible Values (MDY, DMY, YMD)
+   */
+  static getDateFormatOrder(momentDateFormat) {
+    let dateFormatOrder;
+    if (momentDateFormat === 'MM/DD/YYYY') {
+      dateFormatOrder = DateUtil.dateOrder.MDY;
+    } else if (momentDateFormat === 'DD/MM/YYYY') {
+      dateFormatOrder = DateUtil.dateOrder.DMY;
+    } else if (momentDateFormat === 'YYYY/MM/DD') {
```

Comment:
After entering a valid input the auto focus is not being advanced to the next input for the `en-ca`, `sv`, and `sv-se` locales. The reason is because their format should be [YYYY-MM-DD](https://github.com/moment/moment/blob/develop/locale/en-ca.js#L27) instead of with `/` as the separator so the correct `dateFormatOrder` is not being returned.

Reply:
Addressed here : https://github.com/cerner/terra-framework/pull/1208/commits/ef6e9b4856db01f837879b86710bbe32a85e6a66